brave/browser-laptop
# Concat margin properties inside dialog on modalDialog.js

Fix #10138

Auditors:

Test Plan:
1. Open `about:preferences#payments`
2. Click `Add funds`
3. Make sure the modal dialog has margin-top: 50px

Submitter Checklist:


Test Plan:


Reviewer Checklist:

Tests
